Go Language Syntax Support

LoveMHz opened the pull request:
Added initial support for Go language syntax.

Manoz committed:
Add missing css keywords / values

New keywords : animation-delay animation-direction animation-duration animation-iteration-count animation-name background-size background-clip box-sizing overflow-x box-shadow opacity flex-flow resize align-content transform transition

New values : inherit content-box border-box table flex row wrap space-around infinite linear linear-gradient rgba webkit-box scale rotate translate translateX translateY skewX skewY

Manoz committed:
Add ALL css properties (css1, css2 and css3)

Manoz committed:
Less keywords per line

LoveMHz committed:
Inital support Go language syntax

appleboy commented:
@LoveMHz Could you resolve the code conflicts?

LoveMHz commented:
What code conflicts?

Viper007Bond commented:
I don't know why the tests are being dumb and saying there's a conflict. I'll manually merge and push it out.

appleboy commented:
@Viper007Bond Nice work!

appleboy commented on syntaxhighlighter.php:
```diff
@@ -184,6 +185,7 @@ function __construct() {
 			'erl'           => 'erlang',
 			'erlang'        => 'erlang',
 			'fsharp'        => 'fsharp',
+			'go'            => 'go',
```
Please add another syntax `golang`.

LoveMHz replied:
Done

appleboy replied:
👍 💯

LoveMHz committed:
Update syntaxhighlighter.php

Add another syntax golang

LoveMHz commented:
Added golang. Sorry for the delay!

appleboy commented:
@Viper007Bond Any updates?

Viper007Bond commented:
Nope, been busy, sorry. In the middle of trying to move apartments among other things.

Viper007Bond commented:
If you're super impatient and can't wait (sorry!), it's easy to add additional languages to my plugin without modifying it: http://www.viper007bond.com/wordpress-plugins/syntaxhighlighter/adding-a-new-brush-language/

wevsty committed:
update shBrushCpp.js.
support keywords from C++20.
support cpp special string.
update shBrushPython.js
support keyword async and await.

wevsty committed:
support cpp special character

donnchawp committed:
Tighten auto-link regex so it only links https? links.

donnchawp committed:
Merge pull request #2 from wevsty/master

Add more cpp and python grammar keywords

donnchawp commented:
@LoveMHz Still showing conflicts, but I've copied to the Automattic fork of the repo. https://github.com/Automattic/syntaxhighlighter/pull/5

donnchawp committed:
Merge pull request #1 from Automattic/fix/url_regex

Tighten auto-linker regex so it only links https? text

donnchawp committed:
Merge pull request #6 from Manoz/master

Add missing css keywords / values

donnchawp committed:
Update docs, changelog, version to 3.5.1

donnchawp committed:
Merge pull request #7 from Automattic/release_3_5_1

Update docs, changelog, version to 3.5.1

LoveMHz committed:
Merge branch 'master' into feature_golang_syntax

federicoleon commented:
Hi there!! Any news on this PR?? Do we have an ETA for merging this? @Viper007Bond @LoveMHZ ?

pacuna commented:
Please merge this!

alexsanford commented:
@LoveMHz could you please rebase this onto the latest `master`, and ensure it is only making the required changes? I see, for example, several changes to `readme.txt` and other brush files which do not appear to be related to this PR.

If we can get that cleaned up, then we should be able to get this merged 🙂

diegoacuna commented:
@LoveMHz nice work on this, any chances on having this merged any time soon? if you don't have the time maybe we can open a new PR with the required changes on the code

smarquezs commented:
Please merge this!

alexsanford commented:
@diegoacuna please feel free to open a new PR if you would like. This one needs some work, and it seems the most recent interaction from the original author was in 2016. If we can get a PR up and working that only makes the required changes, then we should be able to get it merged 🙂

yscik commented:
Done in #194
